app/scripts/modules/core/src/pipeline/details/SingleExecutionDetails.tsx

```diff
@@ -128,6 +128,9 @@ export class SingleExecutionDetails extends React.Component<
   public render() {
     const { app } = this.props;
     const { execution, sortFilter, stateNotFound } = this.state;
+    let { senderApplication: application, senderExecutionId: executionId } = ReactInjector.$state.params;
```

Comment:
If you pass an object in the params such as `executionParams`, then you could do:
```
const defaultExecutionParams = { application: app.name, executionId: execution && execution.id };
const executionParams = ReactInjector.$state.params.executionParams || defaultExecutionParams;
```
and then use `executionParams` in the sref:
```
<UISref to="^.executions.execution" params={executionParams}>
```